Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.9-buster as base
+# Perform common operations, dependency installation etc...
+
+ENV POETRY_HOME=$HOME/.poetry
+ENV PATH=$POETRY_HOME/bin:$PATH
+RUN echo "--- Installing Poetry ---"
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+RUN echo "--- Installing dependencies ---"
+
+WORKDIR /todo_app
+COPY pyproject.toml poetry.lock /todo_app/
+
+RUN poetry install
+
+COPY ./todo_app /todo_app/todo_app
+
+FROM base as production 
+# Configure for production
+#Implement as CMD, Gunicorn, is it necessary to have brackets in ENTRYPOINT, how to differentiate between ENTRYPOINT & CMD
```

Comment:
It's not necessary to use the brackets syntax for either ENTRYPOINT or CMD, but it's the recommended way.
And if you have both an ENTRYPOINT and a CMD, then you should have the square brackets.

Having only ENTRYPOINT or only CMD is almost functionally the same - the difference is just how you would override it in the `docker run` command. And I'd say if you don't expect people to want to override it, it should be an ENTRYPOINT.